sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -966,7 +966,7 @@ export class BlockBlobClient extends BlobClient {
     getBlockList(listType: BlockListType, options?: BlockBlobGetBlockListOptions): Promise<BlockBlobGetBlockListResponse>;
     stageBlock(blockId: string, body: HttpRequestBody, contentLength: number, options?: BlockBlobStageBlockOptions): Promise<BlockBlobStageBlockResponse>;
     stageBlockFromURL(blockId: string, sourceURL: string, offset?: number, count?: number, options?: BlockBlobStageBlockFromURLOptions): Promise<BlockBlobStageBlockFromURLResponse>;
-    upload(body: HttpRequestBody, contentLength: number, options?: BlockBlobUploadOptions): Promise<BlockBlobUploadResponse>;
+    upload(body: HttpRequestBody | NodeJS.ReadableStream, contentLength: number, options?: BlockBlobUploadOptions): Promise<BlockBlobUploadResponse>;
```

Comment:
Wish to confirm if customer wishes non-retriable or retriable support? If customer wishes non-retirable, this is great! If customer wants retriable support. May be we can buffer bits read from stream?